libs/utils/test.py

```diff
@@ -87,6 +87,14 @@ def runExperiments(cls):
         test_env = TestEnv(test_conf=cls._getTestConf())
 
         experiments_conf = cls._getExperimentsConf(test_env)
+
+        if ITERATIONS_FROM_CMDLINE:
```

Comment:
Why not naming this just ITERATIONS which matches the name used for the experiments_conf attribute?